test/functional/wallet_hd.py

```diff
@@ -48,7 +48,7 @@ def run_test(self):
         # Also send funds to each add
         self.nodes[0].generate(101)
         hd_add = None
-        num_hd_adds = 300
+        num_hd_adds = 10
```

Comment:
nit: Since you touch this anyway and reviewers most likely look at the usage of this var anyway, you might as well make it `NUM_HD_ADDS`, since it is a constant.

Reply:
done. Also touched the second loop, replacing `_` with `i` since it's actually being used.